environments/se/core/src/main/java/org/jboss/weld/environment/se/Weld.java

```diff
@@ -391,6 +430,28 @@ public Weld addExtension(Extension extension) {
         return this;
     }
 
+    @Override
+    public Weld addExtensions(Extension... extensions) {
+        for (Extension extension : extensions) {
+            addExtension(extension);
+        }
+        return this;
+    }
+
+    @Override
+    public Weld addExtensions(Class<? extends Extension>... extensionClasses) {
+        Object[] constructorParams = new Object[] {};
```

Comment:
This variable is useless in most cases. I would rather pass `new Object[] {}` to log method directly.